src/docs/self-hosted/troubleshooting.mdx

```diff
@@ -114,6 +114,14 @@ docker-compose run -T web cleanup --days 7 -m nodestore -l debug
 docker-compose run -T postgres bash -c "apt update && apt install -y --no-install-recommends postgresql-9.6-repack && su postgres -c 'pg_repack -E info -t nodestore_node'"
 ```
 
+If the script above still does not clear up enough disc space, you can try the following in postgres. This will lead to data loss in event details. _SENTRY_RETENTION_DAYS_ will need to be filled in manually.
```

Comment:
```suggestion
If the script above still does not clear up enough disk space, you can try the following in Postgres. This will lead to data loss in event details. _SENTRY_RETENTION_DAYS_ will need to be filled in manually.
```